twake/backend/node/src/services/channels/services/channel/activities/service.ts

```diff
@@ -151,9 +158,14 @@ export default class Activities implements Initializable {
     return this;
   }
 
-  // Log your generic data object
-  logGenericData(event: string, data: GenericObjectType): void {
+  async notify(event: string, data: GenericObjectType): Promise<void> {
     const child = logger.child(data);
```

Comment:
`child` is nice but this is not the way to go here. You can log the data like this:

`logger.debug("Some message %o", data)`

Reply:
Done 👍🏻